components/cascader/style/index.less

```diff
@@ -1,240 +1,102 @@
 @import '../../style/themes/index';
 @import '../../style/mixins/index';
 @import '../../input/style/mixin';
+@import '../../checkbox/style/mixin';
 
 @cascader-prefix-cls: ~'@{ant-prefix}-cascader';
 
+.antCheckboxFn(@checkbox-prefix-cls: ~'@{cascader-prefix-cls}-checkbox');
```

Comment:
rc-cascader has been refactored with the component `select`, we should also refactor `nz-cascader`

Reply:
Will do that.

Reply:
Cool! I'm working on the other components' styling issues, so if you have time you can fix the styles of cascader first, and I think it's better to fix them in style only instead of refactoring it(of course, the premise is that we can do this) I will re-check those components which needs refactoring after all styles work well :)